### DISCUSSION TITLE
Problema con VScode

### DISCUSSION THREAD

**Discussion by EAleman8** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#476] [Q&A] [answered]
Que tal chic@s!

Desde el onboarding se me presentó lo siguiente:

Tengo este problemita con este archivo que siempre que abro el programa aparece como pestaña y no puedo cerrarlo o quitarlo de ninguna forma.
Al intentar cerrarlo, no importa si doy click en "save", o "don't save", se queda ahí y me aparece siempre el recuadro de la esquina inferior derecha:

![imagen](https://user-images.githubusercontent.com/99446604/159212074-e938bbd9-a238-4356-a4fd-c49c5c85d33d.png)

No me provoca ningún inconveniente con los demás archivos u otras cuestiones, pero simplemente quisiera quitarlo y no desaparece.
Alguien podría decirme a qué se debe este problema y/o cómo solucionarlo?

**Comment by hackit92** [comment, marked as answer]
Y si abres una nueva instancia de VSC lo jala? Me suena como a que cuando hiciste las practicas de python se quedó ese archivo anclado y ya no encuentra la ruta, por eso da error desconocido

**Reply by EAleman8** [reply]
Al parecer funcionó, no se me había ocurrido, ya aparece todo limpio al abrir vscode. Gracias!

**Comment by Dfredude** [comment]
Solo tienes que cerrar ese archivo y ya...
VS Code no encuentra el archivo, porque se modificó (movio, eliminó, renombro, etc)

**Reply by EAleman8** [reply]
No me deja cerrarlo, al hacerlo y preguntarme si quiero o no guardar los cambios no importa lo que seleccione, se queda ahí y aparece el error de la esquina
